### ISSUE TITLE
Bug: can you update dependency for vite 3?

### ISSUE THREAD

**Issue by waynebrantley** [Shinigami92/vite-plugin-time-reporter#16]

**Comment by Shinigami92** [comment]
@waynebrantley please test out v1.1.0 and report any issues (I haven't tested myself ^^')
If something broke, please re-open this issue and say what's wrong
Otherwise have a nice day :)

**Comment by waynebrantley** [comment]
@Shinigami92 would you consider adding vite 4 now?

**Comment by Shinigami92** [comment]
There is a breaking change in the warning
https://github.com/Shinigami92/vite-plugin-time-reporter/blob/2cf106d836555888394cfca3d6495c40ce9bcc20/src/index.ts#L44

Therefore I do not have time this evening to investigate

**Comment by waynebrantley** [comment]
@Shinigami92 ok no problem - FYI - I am using the current version with V4 successfully
```
[vite:dts] Start generate declaration files...
┌─────────┬──────────┐
│ (index) │   time   │
├─────────┼──────────┤
│  build  │ '0.5[24] │
│ render  │ '0.093s' │
│  full   │ '0.798s' │
└─────────┴──────────┘
[vite:dts] Declaration files built in 3685ms.
```

**Comment by Shinigami92** [comment]
> @Shinigami92 ok no problem - FYI - I am using the current version with V4 successfully
> 
> ```
> [vite:dts] Start generate declaration files...
> ┌─────────┬──────────┐
> │ (index) │   time   │
> ├─────────┼──────────┤
> │  build  │ '0.5[24] │
> │ render  │ '0.093s' │
> │  full   │ '0.798s' │
> └─────────┴──────────┘
> [vite:dts] Declaration files built in 3685ms.
> ```

That's good to know, but it might fail if in any kind of way an error would be thrown, logging the error as warning now might be broken due to underlying API changes
Or the Error might be converted to a string and might result in e.g. `[object Object]`

**Comment by Shinigami92** [comment]
@waynebrantley to keep you up to date: I reported an issue at https://github.com/rollup/rollup/issues/4754
Feel free to subscribe to that one
I will wait on an answer there, and want to omit to introduce an unneeded breaking change, so that I can still support Vite 2-4 with the same code base

**Comment by Shinigami92** [comment]
@waynebrantley I released 2.1.0
Please report and open new issues when new problems occur
